### PR TITLE
Reduce ttl of persisted final workflow to 1 day

### PR DESCRIPTION
**Pull request by jingzhang36**

### CODE REVIEW

File: manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-deployment.yaml [hunk 1]
```diff
@@ -19,6 +19,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          # If you want a different TTL_SECONDS_AFTER_WORKFLOW_FINISH than the
+          # default 1 day, please specify it here as demonstrated below.
+          # - name: TTL_SECONDS_AFTER_WORKFLOW_FINISH
+          #   value: "172800"
```
Comment (Bobgy) [inline manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-deployment.yaml:25]: Shall we add it to manifest? I think that's better than comment

Comment (rmgogogo) [reply manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-deployment.yaml:25]: and the MKP manifest

Comment (jingzhang36) [reply manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-deployment.yaml:25]: Do you mean adding usage to the README.md in manifests folder or adding the commented configuration in the kustomization.yaml in manifests or adding the commented out configuration directly in the server deployment yaml like here?

I am ok with any. Just want to make sure to be consistent with the conventional approach...

Comment (Bobgy) [reply manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-deployment.yaml:25]: adding the configuration in manifests explicitly, it's a good usage example by itself.
comments are not kept in real deployments, adding it directly has better visibility

Comment (jingzhang36) [reply manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-deployment.yaml]: done

**Pull request merged**